BOSCoreTechnicalWhitePaper.md

```diff
@@ -115,6 +133,31 @@ The advantages of BOSCore cross-chain scheme are as follows:
 
 BOS provides a redemption channel with the EOS main chain based on the IBC scheme. EOS can be easily circulated between the BOS side chain and the EOS main chain, including other high-quality digital certificates on the EOS; similarly, BOS will advance to establish circulation channels with other EOSIO-based sidechains. And the entire EOSIO ecosystem begins to move into an ecological network. BOS will serve as a core circulation link to accelerate the development and evolution of the entire EOSIO ecosystem. 
 
+# Zero Knowledge Proof based Scaling Plan
+
+For the blockchain, TPS affects the carrying capacity of the entire blockchain system, which determines the boundary of the application and is one of the core indicators of the blockchain. In addition to promoting multi-threading and multi-computing area expansion solutions, while considering the implementation of existing smart contracts based on zero-knowledge is a deterministic and finite step, the existing zero-knowledge proof scheme can be improved and typical for finite-step execution. Optimized implementation can meet practical needs, and we will differentiate between different contract features and true computationally intensive contracts using knowledge-proven methods, rather than computationally intensive, using VM execution schemes, which will ultimately improve effectiveness of calculations.
```

Comment:
>and typical for 
the existing zero-knowledge proof scheme can be improved typically for finite-step execution.